### PR TITLE
modelscreated

### PR DESCRIPTION
**Pull request by YunfeiJennifer**
We finish building  User, Item, Booking models!

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Item < ApplicationRecord
+  belongs_to :user
```
Comment (Marionator) [inline app/models/item.rb:2]:
```suggestion
  belongs_to :user
  has_many :bookings
```

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (Marionator) [inline app/models/user.rb:4]:
```suggestion
  has_many :items
  has_many :bookings
  devise :database_authenticatable, :registerable,
```

File: app/models/booking.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :item
```
Comment (jane-doronina) [inline app/models/booking.rb:3]: Group decision not to include belongs_to :user through :item - will use booking.item.user instead

File: app/models/item.rb (hunk 1 above)
Comment (jane-doronina) [inline app/models/item.rb:2]: Item has_many :bookings

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+         :recoverable, :rememberable, :validatable
+end
```
Comment (jane-doronina) [inline app/models/user.rb:6]: has_many :items
has_many :bookings

**Pull request merged**